### DISCUSSION TITLE
Why symfony failed to save the key?

### DISCUSSION THREAD

**Discussion by profesionalaiesam** [symfony/symfony#59030] [Q&A]
### Symfony version(s) affected

6.4

### Description

Hey, why do you think this happens? And how to prevent it?

![image](https://github.com/user-attachments/assets/da67b11d-f00c-41e4-9d89-e326477761ac)

### How to reproduce

I dont know, just happens sometimes in prod

**Comment by HugoSEIGLE** [comment]
Hi, 

Verify the var/cache permission file in your server. Do you use Nginx ?

Then cache clear your app. If the problem persists, you have to do a UMask on php-fpm systemd.

Like this :

```txt
[Service]
UMask=...
```

```shell
php bin/console cache:clear --env=prod
php bin/console cache:warmup --env=prod
```
